src/NLog/LogEventInfo.cs

```diff
@@ -204,7 +204,13 @@ public LogEventInfo(LogLevel level, string loggerName, IFormatProvider formatPro
         /// <summary>
         /// Gets the callsite member function name
         /// </summary>
-        public string CallerMemberName => CallSiteInformation?.GetCallerMemberName(null, false, true, true);
+        [Obsolete("Use CallerMethodName instead. Marked obsolete on NLog 5.0")]
+        public string CallerMemberName => CallSiteInformation?.GetCallerMethodName(null, false, true, true);
```

Comment:
Why the rename? The attribute is called CallerMemberName

Maybe method is better , but I really don't prefer obsoletes

Reply:
The rename is because I find "CallerMethodName" easier to understand. And there are tools like Fody that can also be used to capture caller-context, where they don't use the special attributes.

Reply:
Clear. But because of the obsolete I think we should not do this. It's not that important IMO and obsoletes should be prevented.

So please revert/remove the commit

Reply:
Have reverted the rename of CallerMemberName

Reply:
Thanks!